### ISSUE TITLE
Issue on usage methods

### ISSUE THREAD

**Issue by Cu2ta1n** [Ray7788/CLIP4Minecraft#1]
I see that the code you provided seems to only focus on how to train this CLIP model. How can I use this model as a reward in my work?

**Comment by Ray7788** [comment]
Hi, please check MineDojo official paper https://arxiv.org/abs/2206.08853, which includes MineCLIP part as a reward for the agent, our work is similar to that one

**Comment by Cu2ta1n** [comment]
Okay, thank you for your prompt response. I'll go and look it up now. Due to my limited skills, I may encounter many more problems, so I hope to keep in touch with you! 

Also, you must have used MineCLIP before. When I tested their model, I saw that their results were somewhat unreasonable. The difference between the results of the same text and video calculations was very large. I wonder if you encountered this problem in the first place? I have also raised this [issue](https://github.com/MineDojo/MineCLIP/issues/18) and if you have time, I would greatly appreciate discussing it with you.

**Comment by Ray7788** [comment]
I tested my agents for several times a long time ago, I would say this pre-trained model has limitations understanding complex and new texts(model is small and naive), that's why MineDojo provides some sample texts for demo, if you just use some existing words in the text to drive the agent, it would be more stable.

**Comment by Cu2ta1n** [comment]
Hello, thank you for your reply. The text I am testing here is the `env.task_prompt` provided in minedojo. Then last night during testing, I found that it seemed to be an issue with the loading of model weights? I don't know if you encountered this problem when using it, and how should your model load weights? Is it using `torch.load`? Additionally, there seems to be a `IndentationError` in line 175 of your file `CLIP4MC.py`. Perhaps my question is very basic, but I am extremely grateful for your patience. I also hope to use a more convenient contact method such as WeChat with you.
